common/rpc/encryption/localStorePerHostCertProviderFactory.go

```diff
@@ -67,3 +69,26 @@ func (f *localStorePerHostCertProviderFactory) GetCertProvider(hostName string)
 
 	return cachedCertProvider, nil
 }
+
+func (f *localStorePerHostCertProviderFactory) Expiring(fromNow time.Duration,
```

Comment:
Factory should just create/get instances of other structs. Not doing any business logic.

Reply:
Renamed to *ProviderMap instead.